lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/domain/dva/request/DvaPayload.java

```diff
@@ -53,33 +53,36 @@ public DvaPayload() {}
 
     @JsonCreator
     public DvaPayload(
-            @JsonProperty(value = "surname", required = true) String surname,
-            @JsonProperty(value = "forenames", required = true) List<String> forenames,
+            @JsonProperty(value = "familyName", required = true) String surname,
+            @JsonProperty(value = "givenNames", required = true) List<String> forenames,
             @JsonProperty(value = "dateOfBirth", required = true) LocalDate dateOfBirth,
-            @JsonProperty(value = "issueDate", required = false) LocalDate issueDate,
-            @JsonProperty(value = "dateOfIssue", required = false) LocalDate dateOfIssue,
-            @JsonProperty(value = "expiryDate", required = true) LocalDate expiryDate,
-            @JsonProperty(value = "postcode", required = true) String postcode) {
+            @JsonProperty(value = "validFrom", required = true) LocalDate dateOfIssue,
+            @JsonProperty(value = "validTo", required = true) LocalDate expiryDate,
+            @JsonProperty(value = "driverLicenceNumber", required = true)
+                    String driverLicenceNumber,
+            @JsonProperty(value = "address", required = true) String postcode) {
+        this.requestId = UUID.randomUUID();
         this.surname = surname;
         this.forenames = forenames;
         this.dateOfBirth = dateOfBirth;
-        this.issueDate = issueDate;
         this.dateOfIssue = dateOfIssue;
         this.expiryDate = expiryDate;
+        this.driverLicenceNumber = driverLicenceNumber;
         this.correlationId = UUID.randomUUID();
-        this.requestId = UUID.randomUUID();
         this.timestamp = new SimpleDateFormat(TIMESTAMP_DATE_FORMAT).format(new Date());
         this.postcode = postcode;
     }
+    /*
```

Comment:
@ElliotMurphyGDS I think it will still be useful to have this getter and setter in case we ever need to manually map a DVA payload

Reply:
This was my thinking behind leaving it commented out, we don't need it now but it's there if we ever need in the future